### ISSUE TITLE
Parsing error: Unexpected token, expected "}"

### ISSUE THREAD

**Issue by chenxsan** [mdx-js/eslint-mdx#334]
### Subject of the issue

Here's the content of a mdx file:

<pre>
&lt;div>
  &lt;img />
&lt;/div>

```js
module.exports = {
  plugins: [],
};
```
</pre>

And eslint would report an error shown below:

<img width="605" alt="error" src="https://user-images.githubusercontent.com/1091472/129466985-5581961f-a6b8-4a9a-96cb-b4368555f1be.png">

Now if I prepend a paragraph, e.g.,

<pre>
hello world.

&lt;div>
  &lt;img />
&lt;/div>

```js
module.exports = {
  plugins: [],
};
```
</pre>

Now the error would be fixed.

### Your environment

*   **OS**: 
*   **Packages**: 
*   **Env**: 

### Steps to reproduce

1. clone code from `https://github.com/webpack/webpack.js.org/pull/5298`
2. add a `src/content/mdx.mdx` file with below content:

<pre>
&lt;div>
  &lt;img />
&lt;/div>

```js
module.exports = {
  plugins: [],
};
```
</pre>
3. run `npx eslint src/content/mdx.mdx`

### Expected behaviour

No error.

### Actual behaviour

An error shown.

**Comment by JounQin** [comment]
Thank you, I can reproduce it locally.

**Comment by JounQin** [comment]
@chenxsan Please try

```sh
yarn add https://pkg.csb.dev/mdx-js/eslint-mdx/commit/dc606d44/eslint-plugin-mdx
```

**Comment by chenxsan** [comment]
@JounQin Works fine now.
